quickstarts/microsoft.web/webapp-privateendpoint-vnet-injection/main.bicep

```diff
@@ -0,0 +1,242 @@
+@description('Name of the VNet')
+param virtualNetworkName string = 'vnet1'
+
+@description('Name of the Web Farm')
+param serverFarmName string = 'serverfarm'
+
+@description('Web App 1 name must be unique DNS name worldwide')
+param site1_Name string = 'webapp1-${uniqueString(resourceGroup().id)}'
+
+@description('Web App 2 name must be unique DNS name worldwide')
+param site2_Name string = 'webapp2-${uniqueString(resourceGroup().id)}'
+
+@description('CIDR of your VNet')
+param virtualNetwork_CIDR string = '10.200.0.0/16'
+
+@description('Name of the subnet')
+param subnet1Name string = 'Subnet1'
+
+@description('Name of the subnet')
+param subnet2Name string = 'Subnet2'
+
+@description('CIDR of your subnet')
+param subnet1_CIDR string = '10.200.1.0/24'
+
+@description('CIDR of your subnet')
+param subnet2_CIDR string = '10.200.2.0/24'
+
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+@description('SKU name, must be minimum P1v2')
+@allowed([
+  'P1v2'
+  'P2v2'
+  'P3v2'
+])
+param skuName string = 'P1v2'
+
+@description('SKU size, must be minimum P1v2')
+@allowed([
+  'P1v2'
+  'P2v2'
+  'P3v2'
+])
+param skuSize string = 'P1v2'
+
+@description('SKU family, must be minimum P1v2')
+@allowed([
+  'P1v2'
+  'P2v2'
+  'P3v2'
+])
+param skuFamily string = 'P1v2'
+
+@description('Name of your Private Endpoint')
+param privateEndpointName string = 'PrivateEndpoint1'
+
+@description('Link name between your Private Endpoint and your Web App')
+param privateLinkConnectionName string = 'PrivateEndpointLink1'
+
+var webapp_dns_name = '.azurewebsites.net'
+var privateDNSZoneName = 'privatelink.azurewebsites.net'
+var SKU_tier = 'PremiumV2'
+
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2020-06-01' = {
+  name: virtualNetworkName
+  location: location
+  properties: {
+    addressSpace: {
+      addressPrefixes: [
+        virtualNetwork_CIDR
+      ]
+    }
+  }
+}
+
+resource subnet1 'Microsoft.Network/virtualNetworks/subnets@2020-06-01' = {
+  parent: virtualNetwork
+  name: subnet1Name
+  properties: {
+    addressPrefix: subnet1_CIDR
+    privateEndpointNetworkPolicies: 'Disabled'
+  }
+}
+
+resource subnet2 'Microsoft.Network/virtualNetworks/subnets@2020-06-01' = {
+  parent: virtualNetwork
+  name: subnet2Name
+  properties: {
+    addressPrefix: subnet2_CIDR
+    delegations: [
+      {
+        name: 'delegation'
+        properties: {
+          serviceName: 'Microsoft.Web/serverfarms'
+        }
+      }
+    ]
+    privateEndpointNetworkPolicies: 'Enabled'
+  }
+  dependsOn: [
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties